.github/ISSUE_TEMPLATE.md

```diff
@@ -3,16 +3,32 @@
 
 ### Bug report
 
+**Matplotlib version**
+<!--Please specify your platform and versions of the relevant libraries you are using:-->
+  * Operating system:
+  * Matplotlib version:
+  * Matplotlib backend (`print(matplotlib.get_backend())`):
+  * Python version:
+  * Jupyter version (if applicable):
+  * Other libraries:
+
+<!--Please tell us how you installed matplotlib and python e.g., from source, pip, conda-->
+<!--If you installed from conda, please specify which channel you used if not the default-->
```

Comment:
Do we need that? IMHO it's irrelevant for the majority of reports. We can always ask if it becomes relevant.